script/test_todo.py

```diff
@@ -34,6 +34,19 @@ def verify_task_completed(driver, id):
   assert elem.get_attribute("class") == "completed"
 
 
+def click_button_clear_completed(driver):
+  completed_button = driver.find_element_by_xpath("/html/body/section/div/footer/button")
+  completed_button.click()
+
+
+def verify_all_completed_task_are_deleted(driver):
```

Comment:
This doesn't work.
Please notice that if you don't clear completed items (line 67) this function will still pass.